cms/djangoapps/contentstore/views/block.py

```diff
@@ -9,6 +9,7 @@
 from django.http import Http404, HttpResponse
 from django.utils.translation import gettext as _
 from django.views.decorators.http import require_http_methods
+from cms.djangoapps.contentstore.utils import load_services_for_studio
```

Comment:
Nit: shouldn't this import be in the next section (i.e., above `from cms.lib.xblock.authoring_mixin import VISIBILITY_VIEW`)?

Reply:
yes thanks for catching that